packages/core/admin/admin/src/hooks/useMenu.ts

```diff
@@ -49,7 +50,8 @@ const useMenu = (shouldUpdateStrapi: boolean) => {
           id: 'global.marketplace',
           defaultMessage: 'Marketplace',
         },
-        to: '/marketplace',
+        to: 'https://market.strapi.io',
+        target: '_blank',
```

Comment:
Seeing as the marketplace is an external link now, do we need to maintain any
marketplace admin permissions? e.g. all references to:
`admin::marketplace.read`